README.md

```diff
@@ -185,7 +186,6 @@ group.background {
 group.utility {
     // Run on utility queue, in parallel to the previous block
 }
-group.wait()
```

Comment:
@eneko I prefer the shorter way which is conciser . But I do agree with you. 
@duemunk  What do you think about it?
I think have `notifyInMain`, `notifyInBackground`, etc. and `notifyInQueue(queue:)` instead of just `notify` is better. Thanks for your advise @eneko .